python-crud-google-spreadsheet/README.md

```diff
@@ -0,0 +1,50 @@
+# Python3 CRUD + Google Spreadsheet Integration With Functions
+Create, Read, Update, and Delete entries using a google spreadsheet for storage
+
+# What does it do?
+Creates a simple crud service using only binaris functions written in python3.
+
+# Deploy me!
+
+[Get a Binaris API Key](https://binaris.com/) and a google [project + credentials](https://gspread.readthedocs.io/en/latest/oauth2.html) and a usable [spreadsheet](https://sheets.new) from Google Docs.
+
+```bash
+# Install the Binaris CLI
+$ npm install -g binaris
+
+# Set your API key in ~/.binaris.yml
+$ bn login
+
+# Clone this repo
+$ git clone git@github.com:binaris/functions-examples.git
+$ cd python-crud-google-spreadsheet
+
+# After cloning the repo, move your credentials into the repo working directory
+$ cp {path-to-credentials} ./credentials.json
```

Comment:
What credentials?

Reply:
Referenced earlier in the readme, it's required from the Google api account.